server/src/operators/organization_operator.rs

```diff
@@ -622,3 +621,29 @@ pub async fn delete_actual_organization_query(
 
     Ok(())
 }
+
+pub async fn update_all_org_dataset_configs_query(
+    org_id: uuid::Uuid,
+    new_config: serde_json::Value,
+    pool: web::Data<Pool>,
+) -> Result<(), ServiceError> {
+    let concat_configs_raw_query = sql_query(format!(
+        "UPDATE datasets SET server_configuration = server_configuration || '{}' WHERE organization_id = '{}';",
+        new_config, org_id
+    ));
+
+    let mut conn = pool.get().await.unwrap();
+
+    concat_configs_raw_query
+        .execute(&mut conn)
+        .await
+        .map_err(|e| {
+            log::error!(
+                "Error updating datasets in update_all_org_dataset_server_configs: {:?}",
+                e
+            );
+            ServiceError::BadRequest("Error updating datasets".to_string())
+        })?;
+
+    Ok(())
+}
```

Comment:
After making this change we are down from 20s to 200ms: 

![image](https://github.com/user-attachments/assets/4d6f6871-0646-40ea-9087-027ecdadc141)

Reply:
What were you doing before? The commit got force pushed over